### PR TITLE
Feature/23 linters

### PR DESCRIPTION
**Pull request by dilyara4949**
Add linter configs to the project and fix errors

### CODE REVIEW

File: internal/handler/api.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func NewAPI(cfg config.Config, database *gorm.DB) *gin.Engine {
 			}
 			users := v1.Group("/users")
 			{
-				admin := users.Use(middleware.JWTAuth(cfg.JWTTokenSecret), middleware.AccessCheck(adminRole))
+				admin := users.Use(middleware.JWTAuth(cfg.JWTTokenSecret), middleware.AccessCheck("admin"))
```
Comment (krylphi) [inline internal/handler/api.go:37]: Why was it changed from the constant back to string?

**Review by krylphi — COMMENTED** [review]
One small comment. Otherwise looks OK, but I'm not sure that we need all of the linters enabled.

File: internal/handler/api.go (hunk 1 above)
Comment (dilyara4949) [reply internal/handler/api.go]: i got confused before, i was mistake

**Review by krylphi — APPROVED** [review]
LGTM

**Pull request merged**